docs/getting-started/dockstore-workflows.rst

```diff
@@ -165,10 +159,6 @@ Now that your workflow has been added, any time there is a push to a branch on G
 it is automatically updated on Dockstore! Anytime there is a deletion of a branch on GitHub that has a ``/.dockstore.yml``, the version is
 removed from Dockstore.
 
-.. tip:: Since the workflows field is an array, this file supports multiple workflows on Dockstore stemming from
```

Comment:
Why did you remove this? Not saying it was wrong to, but just wondering.

Reply:
Because of the other elements right under it, this renders as multiple colored boxes stacked on top of each other. It's a bit distracting, and since this gives information that isn't very important, it shouldn't get its own box in a place where there's already boxes. I moved its information to line 135.